app/src/main/java/com/quran/labs/androidquran/ui/translation/TranslationView.java

```diff
@@ -66,8 +74,38 @@ public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
     ayahToolBar = new AyahToolBar(context, R.menu.share_menu);
     ayahToolBar.setOnItemSelectedListener(this);
     ayahToolBar.setVisibility(View.GONE);
-    addView(ayahToolBar, LayoutParams.WRAP_CONTENT,
-        context.getResources().getDimensionPixelSize(R.dimen.toolbar_total_height));
+    addView(ayahToolBar, LayoutParams.WRAP_CONTENT, context.getResources().getDimensionPixelSize(R.dimen.toolbar_total_height));
+
+    ImageButton copyButton = (ImageButton) ayahToolBar.findViewById(R.id.cab_copy_ayah_text_menu);
+    copyButton.setOnClickListener(new View.OnClickListener() {
+
+      @Override
+      public void onClick(View v) {
+        PopupMenu popmenu = new PopupMenu(context, v);
+        int items = translations.length;
+        String[] titles = new String[items];
+
+        for (int i = 0; i < items; i++) {
+          popmenu.getMenu().add(Menu.NONE, i, 1, translations[i]);
```

Comment:
Please use `Menu.NONE` for the order (3rd argument) as well. 

Also, I wonder if it'd be wise to cache the return value of `popmenu.getMenu()`...